decidim-admin/app/helpers/decidim/admin/attributes_display_helper.rb

```diff
@@ -31,7 +31,14 @@ module AttributesDisplayHelper
       # attrs - a list of N attributes of the `record`.
       def display_for(record, *attrs)
         attrs.map do |attr|
-          display_label(record, attr) + display_value(record, attr)
+          if record.column_for_attribute(attr).type == :hstore
+            I18n.available_locales.map do |locale|
+              content_tag(:dt, record.class.human_attribute_name(attr) + " (#{locale})") +
```

Comment:
Currently the forms show i18n attributes in the form `Title (en)`, instead of the actual translation of the attribute name, so this is following the same pattern. This is done to avoid the need to know the word in the given language.